File: 백준/Bronze/10818. 최소， 최대/최소， 최대.py
#풀이1

# a = int(input())

# list = list(map(int,input().split()))

# print(min(list),max(list))

#풀이2

a = int(input())

list = list(map(int,input().split()))

min = list[0]
max = list[0]


for x in range(a): 
    
    if min<list[x]:
        min = min
    else:
        min = list[x]

for x in range(a):
    if max>list[x]:
        max = max
    else:
        max = list[x]    

print(min,max)

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.*;
import java.util.stream.Collectors;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        sc.nextLine();
        
        String[] str = sc.nextLine().split(" ");
        List<Integer> list = Arrays.stream(str)
            .map(s ->Integer.parseInt(s))
            .collect(Collectors.toList());
        
        int max = Collections.max(list);
        int min = Collections.min(list);
        System.out.println(min + " " + max);
    }
}